qiskit/visualization/counts_visualization.py

```diff
@@ -87,6 +74,14 @@ def plot_histogram(
     Args:
         data (list or dict): This is either a list of dictionaries or a single
             dict containing the values to represent (ex ``{'001': 130}``)
+            Note: Passing `QuasiDistribution`, `ProbDistribution`, or a distribution dictionary
+            to the `data` argument is deprecated.
+
+            If you would like to plot a histogram from a `QuasiDistribution`,
+            `ProbDistribution`, or a distribution dictionary you should be using the
+            :func:`.plot_distribution` function.
```

Comment:
If it's been properly deprecated, do we still need this description? It might be better to just document this in the reno